purpleair-aqi.js

```diff
@@ -486,8 +481,12 @@ async function run() {
   const listWidget = new ListWidget();
   listWidget.useDefaultPadding();
 
-try{
-  const sensorId = await getSensorId();
+  try {
+    if (!API_KEY) {
+      throw `You need a PurpleAir API Key for this widget.`;
```

Comment:
This was all that we need to show that new error state.
It's red text on a white background.